environment.default

```diff
@@ -0,0 +1,6 @@
+# Default environment variables for cellxgene development
+set -a
```

Comment:
I'm unsure why you need these `set` mode changes?   As far as I can tell, you don't rely on this file being executed by bash (which would understand them) or imported into a Makefile except via the `sed` command in common.mk (which ignores them).

So they seem unnecessary.  If they are necessary, the need and purpose is somewhat opaque and needs a comment.

Reply:
The ideas here are the following:
1. If you were to rely on `source`-ing the environment directly, `set -a` will propagate theses variables to subprocesses. The variables in the environment file are then authoritative.
2. If people use this file as a template and create their own environments--say `environment.colin`-- they could `source` it and the changes would be propagated as suggested in (1)

Reply:
I agree that is what happens when you have bash execute the file.  However, you are not doing that -- you are "grepping" through the file from common.mk.   Unless I'm completely confused, these two lines have zero effect on the execution environment, and can be safely removed.

Reply:
I guess the primary purpose here was to set an example for other people who were creating their own environment files. In this case you need it, since `common.mk` pulls from `environment.default` directly. Point taken, though. I will remove them.